### DISCUSSION TITLE
Starting the project locally

### DISCUSSION THREAD

**Discussion by markmadlangbayan** [cribeiro84/azure-devops-pull-request-hub#234] [Q&A] [answered]
Hello, I'm trying to start the project locally.  So, I did the following:

1. Cloned the repo
2. npm install
3. HTTPS=true npx react-scripts start

Chrome opens https://localhost:3000.  I then instruct to run it anyway even though there's no SSL cert.

Now I just see a white screen with a loading indicator in the middle.

Are there additional steps that I need to do to check it out locally?

**Comment by cribeiro84** [comment, marked as answer]
Hey @markmadlangbayan, you need first to create an extension profile at Azure Marketplace, build a dev version of the extension (npm run package-dev), upload it to the marketplace and access it from there. The extension will be installed on your Azure DevOps org and it will redirect you to localhost:3000 address for debugging.

**Reply by markmadlangbayan** [reply]
Thanks @cribeiro84!
